app/css/base.css

```diff
@@ -1,6 +1,6 @@
 @font-face {
   font-family: "gaia-icons";
-  src: url("../media/fonts/gaia-icons.ttf") format("truetype");
+  src: url("../media/fonts/gaia-icons.woff") format("truetype");
```

Comment:
WOFF works on every FxOS version. And we can see the correct font for the text. I think I just deleted the icon fonts too.